frontend/public/components/about-modal.tsx

```diff
@@ -38,39 +31,27 @@ import {
 const DynamicPlugins: React.FC = () => {
   const { t } = useTranslation();
   const [pluginInfoEntries] = useDynamicPluginInfo();
-  const [rows, setRows] = React.useState([]);
+  const [items, setItems] = React.useState([]);
 
   React.useEffect(() => {
     const loadedPlugins = pluginInfoEntries.filter(isLoadedDynamicPluginInfo);
-    setRows(
-      loadedPlugins?.map((plugin) => {
-        return {
-          cells: [plugin.metadata.name, plugin.metadata.version],
-        };
+    const sortedLoadedPlugins = loadedPlugins.sort((a, b) =>
```

Comment:
lets check if any plugins are loaded
```suggestion
    if loadedPlugins.length > 0 {
        const sortedLoadedPlugins = loadedPlugins.sort((a, b) =>
        ...
```

Reply:
Assuming I understand your comment correctly and you want me to nest lines 38-50 inside the if statement, this will introduce a bug if one or more `loadedPlugins` exist, the modal is rendered, and then `loadedPlugins` goes to zero as `items` will never get set back to `[]` and the modal will continue to display `loadedPlugins` that are no longer loaded.

Reply:
^ @jhadvig